App/arca/components/HelpCenter.jsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { useState } from 'react';
+import "../assets/css/components.scss";
+import "../assets/css/main.scss";
+import "../assets/css/helpCenter.scss";
+import HelpCenterBody from "./HelpCenterBody";
+
+function HelpCenter() {
+  const [itemSelected, setItemSelected] = useState(false);
+  const [lastItemSelected, setLastItemSelected] = useState("");
+  const [sectWithOut, setSectWithOut] = useState(true);
+  const [lastSectSelected, setLastSectSelected] = useState("");
+
+  function selectItem(classItemS, numberClass) {
+    if (!itemSelected) {
+      let item = document.querySelector(classItemS);
+      item.classList.add("item-help-selected");
```

Comment:
Avoid updating the DOM using vanilla javascript, this kind of change is the purpose of React.js.

You can store the ID selected and add the class to the element using a ternary operator.


```suggestion
      const [idItemSelected, setIdItemSelected] = useState("");
      ...
      const selectItem = (idItem) => {
          setIdItemSelected(idItem)
      }
      ...
      <div 
          className={`item-help ${idItemSelected === 'item-1' ? 'item-help-selected' : undefined}`}
          onClick={() => selectItem('item-1')}
      ...
      
      
```